tsconfig.json

```diff
@@ -42,7 +42,7 @@
     "noFallthroughCasesInSwitch": true /* Report errors for fallthrough cases in switch statement. */,
 
     /* Module Resolution Options */
-    // "moduleResolution": "node",            /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */
+    "moduleResolution": "node",            /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */
```

Comment:
why is this change needed?

Reply:
Without it I get:

```
src/omit.test.ts:1:28 - error TS2792: Cannot find module 'ts-expect'. Did you mean to set the 'moduleResolution' option to 'node', or to add aliases to the 'paths' option?

1 import { expectType } from 'ts-expect';
```

This being said, `expectType` is such a small function, we could define it ourselves and skip that dependency altogether, if we want.